dot/core/messages.go

```diff
@@ -16,13 +17,13 @@ import (
 )
 
 func (s *Service) validateTransaction(peerID peer.ID, head *types.Header, rt runtime.Instance,
-	tx types.Extrinsic, toPropagate *[]types.Extrinsic) (bool, error) {
+	tx types.Extrinsic) (validity *transaction.Validity, failed bool, err error) {
```

Comment:
failed is meant to be `ok` or `success` here right?

```suggestion
	tx types.Extrinsic) (validity *transaction.Validity, valid bool, err error) {
```

Reply:
yup good call!